crates/biome_configuration/src/linter/mod.rs

```diff
@@ -219,3 +220,65 @@ pub struct RuleWithOptions<T: Default> {
     pub level: RulePlainConfiguration,
     pub options: T,
 }
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+pub struct RuleCode {
+    pub group: &'static str,
+    pub name: &'static str,
+}
+
+impl<'a> From<RuleCode> for RuleFilter<'a> {
+    fn from(RuleCode { group, name }: RuleCode) -> Self {
+        RuleFilter::Rule(group, name)
+    }
+}
+
+impl FromStr for RuleCode {
+    type Err = &'static str;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if let Some((group, rule_name)) = s.split_once('/') {
+            if let Some((group, name)) = Rules::matches_diagnostic_code(group, rule_name) {
+                Ok(RuleCode { group, name })
+            } else {
+                Err("the rule doesn't exist.")
+            }
+        } else {
+            Err("a group and a rule name separated by a slash is required.")
```

Comment:
Did you start the phrase with a lower case letter on purpose? e.g. `the rule...` VS `The rule...`

Reply:
Because it is displayed after a double colon `:` in bpaf. I didn't check if we use another convention in other errors.